test/unit/services/awsAdapter_spec.js

```diff
@@ -5,7 +5,9 @@ var expect     = require("chai").expect;
 var Bluebird   = require("bluebird");
 var Sinon      = require("sinon");
 var AWS        = require("aws-sdk");
-var ec2        = new AWS.EC2();
+var ec2        = new AWS.EC2({
```

Comment:
This should probably happen inside a `describe` block since it won't get garbage collected otherwise.